rpmlb/cli.py

```diff
@@ -126,3 +135,15 @@ def run(recipe_file, collection_id, **option_dict):
     builder.run(work, **option_dict)
 
     log.info('Success!')
+
+
+def _adjust_option_dict(option_dict):
```

Comment:
This is a very generic name. Lets call it `_choose_pkg_cmd` or something like that.